### PR TITLE
Link to new watchlist docs

### PR DESCRIPTION
**Pull request by marcovandeveen**
## Description
Link to new watchlist docs publish by doc team.

### Steps completed:
According to our definition of done, I have completed the following steps:

- [ ] User stories met
- [ ] Storybook stories
- [ ] Unit tests added
- [ ] Linting passing
- [ ] Unit tests passing
- [ x] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: docs/features/user-watchlists.md [hunk 1]
```diff
@@ -69,8 +69,13 @@ To ensure a **cross-device experience**, we standardize on the following datafor
 
 ## Watchlist playlist
 
+<<<<<<< HEAD
```
Comment (budzworthy) [inline docs/features/user-watchlists.md]: It looks like the merge has some merge conflict artifacts

**Review by budzworthy — CHANGES_REQUESTED** [review]

File: docs/features/user-watchlists.md (hunk 1 above)
Comment (marcovandeveen) [reply docs/features/user-watchlists.md]: Fixed it!

Comment (budzworthy) [reply docs/features/user-watchlists.md]: I still see 2 instances of "The media metadata..." line as well as a line that says:

```> > > > > > > > f949d534c5d8a7e25032b20b4ff74f74ec6ce453```

Comment (marcovandeveen) [reply docs/features/user-watchlists.md]: :-( I guess I'm missing some understanding of the Git(hub) workflow here.

**Review by budzworthy — APPROVED** [review]

**Pull request merged**